### PR TITLE
refactor: docs/references/sdk for kaia-docs

### PR DESCRIPTION
**Pull request by Sotatek-TanHoang**
## Proposed changes

- Rename docs/references/sdk for kaia-docs
- Example codes, externals links is **NOT** included in this commit and will be added later.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia-docs/blob/main/CONTRIBUTING.md)
- [x] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/references/sdk/caver-java/caver-java.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
 
 ![](/img/references/klaytnXcaver-java.png)
 
-`caver-java` is a lightweight, high modular, convenient Java and Android library to interact with clients \(nodes\) on the Klaytn network: This library is an interface which allows Java applications to easily communicate with [Klaytn](https://www.klaytn.com) network.
+`caver-java` is a lightweight, high modular, convenient Java and Android library to interact with clients \(nodes\) on the Kaia network: This library is an interface which allows Java applications to easily communicate with [Kaia](https://www.klaytn.com) network.
```
Comment (kjeom) [inline docs/references/sdk/caver-java/caver-java.md:5]: You can use the kaia link (https://kaia.io)

File: docs/references/sdk/caver-js/api/caver-kct/kip17.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
 # caver.kct.kip17
 
-`caver.kct.kip17` helps you easily handle a smart contract that implements [KIP-17](https://kips.klaytn.foundation/KIPs/kip-17) as a JavaScript object on the Klaytn blockchain. 
+`caver.kct.kip17` helps you easily handle a smart contract that implements [KIP-17](https://kips.klaytn.foundation/KIPs/kip-17) as a JavaScript object on the Kaia blockchain.
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-kct/kip17.md:3]: Please use https://kips.kaia.io/KIPs/kip-17

File: docs/references/sdk/caver-js/api/caver-kct/kip17.md [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
 
 The `caver.kct.kip17` inherits [caver.contract](../caver.contract.md) to implement the KIP-17 token contract. The `caver.kct.kip17` holds the same properties of `caver.contract` whereas there are additional methods to implement extra features. This section only introduces the newly added bound methods of the `caver.kct.kip17`.
 
-The code that implements KIP-17 for caver-js is available on the [Klaytn Contracts Github Repo](https://github.com/klaytn/klaytn-contracts/tree/master/contracts/KIP/token/KIP17). KIP-17 for caver-js supports Ownable interface. Using this, you can designate a contract owner when deploying a contract
+The code that implements KIP-17 for caver-js is available on the [Kaia Contracts Github Repo](https://github.com/kaiachain/kaia-contracts/tree/master/contracts/KIP/token/KIP17). KIP-17 for caver-js supports Ownable interface. Using this, you can designate a contract owner when deploying a contract
 
-For more information about KIP-17, see [Klaytn Improvement Proposals](https://kips.klaytn.foundation/KIPs/kip-17).
+For more information about KIP-17, see [Kaia Improvement Proposals](https://kips.klaytn.foundation/KIPs/kip-17).
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-kct/kip17.md:9]: please use https://kips.kaia.io/KIPs/kip-17

File: docs/references/sdk/caver-js/api/caver-kct/kip37.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 
 The `caver.kct.kip37` inherits [caver.contract](../caver.contract.md) to implement the KIP-37 token contract. The `caver.kct.kip37` holds the same properties of `caver.contract` whereas additional methods are implemented for extra features. This section only introduces the newly added methods of the `caver.kct.kip37`.
 
-The code that implements KIP-37 for caver-js is available on the [Klaytn Contracts Github Repo](https://github.com/klaytn/klaytn-contracts/tree/master/contracts/KIP/token/KIP37). KIP-37 for caver-js supports Ownable interface. Using this, you can designate a contract owner when deploying a contract
+The code that implements KIP-37 for caver-js is available on the [Kaia Contracts Github Repo](https://github.com/kaiachain/kaia-contracts/tree/master/contracts/KIP/token/KIP37). KIP-37 for caver-js supports Ownable interface. Using this, you can designate a contract owner when deploying a contract
 
-For more information about KIP-37, see [Klaytn Improvement Proposals](https://kips.klaytn.foundation/KIPs/kip-37).
+For more information about KIP-37, see [Kaia Improvement Proposals](https://kips.klaytn.foundation/KIPs/kip-37).
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-kct/kip37.md:9]: https://kips.kaia.io/KIPs/kip-37

File: docs/references/sdk/caver-js/api/caver-kct/kip7.md [hunk 5]
```diff
@@ -1,30 +1,30 @@
...
 
 The `caver.kct.kip7` inherits [caver.contract](../caver.contract.md) to implement the KIP-7 token contract. The `caver.kct.kip7` holds the same properties of `caver.contract` whereas additional methods to implement extra features. This section only introduces the newly added bound methods of the `caver.kct.kip7`.
 
 The abi and bytecode used in the caver.kct.kip7 were implemented using the example of [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/token/ERC20).
 
-The code that implements KIP-7 for caver-js is available on the [Klaytn Contracts Github Repo](https://github.com/klaytn/klaytn-contracts/tree/main/contracts/KIP/token/KIP7).
+The code that implements KIP-7 for caver-js is available on the [Kaia Contracts Github Repo](https://github.com/kaiachain/kaia-contracts/tree/main/contracts/KIP/token/KIP7).
 
-For more information about KIP-7, see [Klaytn Improvement Proposals](https://kips.klaytn.foundation/KIPs/kip-7).
+For more information about KIP-7, see [Kaia Improvement Proposals](https://kips.klaytn.foundation/KIPs/kip-7).
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-kct/kip7.md:11]: https://kips.kaia.io/KIPs/kip-7

File: docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md [hunk 6]
```diff
@@ -227,7 +227,7 @@ NOTE: As of caver-js [v1.8.1-rc.4](https://www.npmjs.com/package/caver-js/v/1.8.
 | feePayer | string | (optional) The address of fee payer. |
 | nonce | string | (optional) A value used to uniquely identify a sender’s transaction. If omitted, `caver.rpc.klay.getTransactionCount(address, 'pending')` will be used to set nonce. |
 | gasPrice | string | (optional) A multiplier to get how much the sender will pay in tokens. If omitted, `caver.rpc.klay.getGasPrice` will be used to set gasPrice. |
-| chainId | string | (optional) The chain id of the Klaytn. If omitted, `caver.rpc.klay.getChainId` will be used to set chainId. |
+| chainId | string | (optional) The chain id of the Kaia. If omitted, `caver.rpc.klay.getChainId` will be used to set chainId. |
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md:230]: not Kaia, kaia

File: docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md [hunk 7]
```diff
@@ -285,13 +285,13 @@ NOTE: As of caver-js [v1.8.1-rc.4](https://www.npmjs.com/package/caver-js/v/1.8.
...
 | gas | string | The maximum amount of transaction fee the transaction is allowed to use. |
-| value | string | (optional, default: `'0x0'`) The amount of KLAY in peb to be transferred. You can use `caver.utils.toPeb`. |
+| value | string | (optional, default: `'0x0'`) The amount of KAIA in peb to be transferred. You can use `caver.utils.toPeb`. |
 | signatures | Array | (optional) An array of signatures. |
 | feePayerSignatures | Array | (optional) An array of feePayerSignatures. |
 | feePayer | string | (optional) The address of fee payer. |
 | nonce | string | (optional) A value used to uniquely identify a sender’s transaction. If omitted, `caver.rpc.klay.getTransactionCount(address, 'pending')` will be used to set nonce. |
 | gasPrice | string | (optional) A multiplier to get how much the sender will pay in tokens. If omitted, `caver.rpc.klay.getGasPrice` will be used to set gasPrice. |
-| chainId | string | (optional) The chain id of the Klaytn. If omitted, `caver.rpc.klay.getChainId` will be used to set chainId. |
+| chainId | string | (optional) The chain id of the Kaia. If omitted, `caver.rpc.klay.getChainId` will be used to set chainId. |
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md:294]: Not Kaia, kaia

File: docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md [hunk 8]
```diff
@@ -351,7 +351,7 @@ NOTE: As of caver-js [v1.8.1-rc.4](https://www.npmjs.com/package/caver-js/v/1.8.
 | feePayerSignatures | Array | (optional) An array of feePayerSignatures. |
 | feePayer | string | (optional) The address of fee payer. |
 | gasPrice | string | (optional) A multiplier to get how much the sender will pay in tokens. If omitted, `caver.rpc.klay.getGasPrice` will be used to set gasPrice. |
-| chainId | string | (optional) The chain id of the Klaytn. If omitted, `caver.rpc.klay.getChainId` will be used to set chainId. |
+| chainId | string | (optional) The chain id of the Kaia. If omitted, `caver.rpc.klay.getChainId` will be used to set chainId. |
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md:354]: Not Kaia, kaia

File: docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md [hunk 9]
```diff
@@ -408,7 +408,7 @@ NOTE: As of caver-js [v1.8.1-rc.4](https://www.npmjs.com/package/caver-js/v/1.8.
...
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md:411]: Not Kaia, kaia

File: docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md (hunk 8 above)
Comment (kjeom) [reply docs/references/sdk/caver-js/api/caver-transaction/fee-delegation.md:354]: Please check all the capital first letter

File: docs/references/sdk/caver-js/api/caver-wallet/keyring.md [hunk 10]
```diff
@@ -330,7 +330,7 @@ SingleKeyring {
     _key: PrivateKey { _privateKey: '0x{private key}' }
 }
 
-// Create singleKeyring from KlaytnWalletKey
+// Create singleKeyring from KaiaWalletKey
```
Comment (kjeom) [inline docs/references/sdk/caver-js/api/caver-wallet/keyring.md:333]: Actually we have no "KaiaWalletKey" in caver. Please rollback

**Comment by Sotatek-TanHoang** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Sotatek-TanHoang** [comment]
recheck

**Review by scott-klaytn — APPROVED** [review]

**Review by ayo-klaytn — APPROVED** [review]

**Pull request merged**